### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by pragex**

### CODE REVIEW

File: rest_registration/api/serializers.py [hunk 1]
```diff
@@ -154,7 +159,7 @@ def get_fields(self):
     def validate(self, attrs):
         if self.has_password_confirm:
             if attrs['password'] != attrs['password_confirm']:
-                raise ValidationError('Passwords don\'t match')
+                raise ValidationError(_('Passwords don\'t match'))
```
Comment (apragacz) [inline rest_registration/api/serializers.py]: A minor thing; you could consider improving existing code by replacing all `'` to `"` for each message in places where you're changing the code so we could have a better consistency.

File: rest_registration/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,108 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (apragacz) [inline rest_registration/locale/fr/LC_MESSAGES/django.po]: I think we should update the metadata here - this looks like auto-generated stuff; you could put yourself as the author of the translation.

File: rest_registration/locale/fr/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+#: rest_registration/utils/verification.py:18
+msgid "Signature expired"
+msgstr "Signature expirée"
+
+#: rest_registration/utils/verification.py:20
+msgid "Invalid signature"
+msgstr "Signature invalide"
+
+#~ msgid "email address"
```
Comment (apragacz) [inline rest_registration/locale/fr/LC_MESSAGES/django.po]: Any idea what's the commented definitions doing here? If they are not needed perhaps they should be removed?

File: rest_registration/locale/fr/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 2.0.6\n"
+
+#: rest_registration/api/serializers.py:162
```
Comment (apragacz) [inline rest_registration/locale/fr/LC_MESSAGES/django.po]: Looks that a lot of work was done there 👍 
Unfortunately I don't know French, so I'll have to assume that the translations are correct ;)

**Comment by apragacz** [comment]
Hi @pragex, Do you have any further comments? do you plan to do some changes in this PR?

**Comment by pragex** [comment]
Hi @apragacz , You have reasons in your comments. I will do the suggested modifications made.

File: rest_registration/api/serializers.py [hunk 5]
```diff
@@ -12,6 +12,11 @@
     get_user_setting
 )
 
+try:
+    from django.utils.translation import ugettext_lazy as _
+except ImportError:
+    def _(text):
```
[the remark this answers is not in the document]
Comment (pragex) [reply rest_registration/api/serializers.py]: I checked in the django sources. The gettext function family are in the django sources. There is no need to check if the library is imported.

**Comment by pragex** [comment]
> Hi @apragacz , You have reasons in your comments. I will do the suggested modifications made.

I think I have fixed all raised.

**Comment by apragacz** [comment]
Hi @pragex ,
thanks again for the contribution!

**Pull request merged**